server/storage/backend/read_tx.go

```diff
@@ -74,7 +76,15 @@ func (baseReadTx *baseReadTx) UnsafeForEach(bucket Bucket, visitor func(k, v []b
 	return baseReadTx.buf.ForEach(bucket, visitor)
 }
 
+func bytesToRev(bytes []byte) int64 {
```

Comment:
The name is a little confusing, because it has the same name but different logic as other implementations, such as [revision.go#L56](https://github.com/etcd-io/etcd/blob/4019c592eabcf4090d26ebd7c1b59ad3172bd890/server/storage/mvcc/revision.go#L56).

So suggest to rename it to something like `getMainRevFromBytes`.